File: app/src/main/java/com/jakubokrasa/uprightchallenge/ui/SettingsActivity.java
package com.jakubokrasa.uprightchallenge.ui;

import androidx.appcompat.app.AppCompatActivity;
import androidx.preference.PreferenceManager;

import android.os.Bundle;

import com.jakubokrasa.uprightchallenge.R;

public class SettingsActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        getSupportFragmentManager().beginTransaction()
                .replace(android.R.id.content, new SettingsFragment())
                .commit();
        PreferenceManager.setDefaultValues(this, R.xml.preferences, false);
    }
}

File: app/src/main/java/com/jakubokrasa/uprightchallenge/ui/SettingsActivity.kt
package com.jakubokrasa.uprightchallenge.ui

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import androidx.preference.PreferenceManager
import com.jakubokrasa.uprightchallenge.R

class SettingsActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        supportFragmentManager.beginTransaction()
                .replace(android.R.id.content, SettingsFragment())
                .commit()
        PreferenceManager.setDefaultValues(this, R.xml.preferences, false)
    }
}